File: October Dusk 2020/Adit's Survey/adit_survey.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        
        int[] bill = new int[n];
        
        for(int i=0;i<n;i++)
        {
            int a = sc.nextInt();
            a *= 8;
            if(a<250)
                bill[i] = 250;
            else
                bill[i] = a;
        }
        
        for(int i=0;i<n;i++)
        {
            System.out.print(bill[i] + " ");
        }
        
    }
}

File: October Dusk 2020/Adit's Survey/adit_survey.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int n;
    cin>>n;
    while(n--)
    {
        int data;
        cin>>data;
        int ans = data*8;
        if(ans < 250)
            cout<<250<<" ";
        else
            cout<<ans<<" ";
    }
    return 0;
}